pwnlib/commandline/libcdb.py

```diff
@@ -89,15 +91,15 @@
 hash_parser.add_argument(
     '--unstrip',
     action = 'store_true',
-    default = True,
+    default = False,
```

Comment:
Okey, but I believe it would be better to set the default value of `--no-unstrip` to `False`. This way, I can pass `not args.no_unstrip` as the value for `unstrip`.

In the following code, by default, `unstrip` is `True`. If we use the `--no-unstrip`, the `unstrip` parameter will become `False`.

```python
symbols = {pairs[i]:pairs[i+1] for i in range(0, len(pairs), 2)}
matched_libcs = libcdb.search_by_symbol_offsets(symbols, raw=True, unstrip=not args.no_unstrip, offline=args.offline)
for libc in matched_libcs:
    print_libc(libc)
    fetch_libc(args, libc)
```